strawberryfields/ops.py

```diff
@@ -1349,14 +1302,15 @@ class BSgate(Gate):
     """
     ns = 2
 
-    def __init__(self, theta=pi/4, phi=0.):
+    def __init__(self, theta=np.pi/4, phi=0.):
         # default: 50% beamsplitter
         super().__init__([theta, phi])
 
     def _apply(self, reg, backend, **kwargs):
-        t = cos(self.p[0])
-        r = sin(self.p[0]) * exp(1j * self.p[1])
-        backend.beamsplitter(t.x, r.x, *reg)
+        p = par_evaluate(self.p)
+        tr = (np.cos(p[0]),
```

Comment:
I don't really see the benefit of making the tuple here. Suggest reverting back to two variables

Reply:
Concur. It adds a layer of extra complexity when reading the code.

Reply:
Right